transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -58,6 +59,23 @@ public static synchronized HyperwalletTransferMethodUi getInstance(
         return sInstance;
     }
 
+    /**
+     * Returns an instance that will also initialize the Insights library.
+     *
+     * @param context                     A Context of the application consuming this Intent.
+     * @param authenticationTokenProvider An implementation of the {@link HyperwalletAuthenticationTokenProvider}
+     * @return singleton instance of HyperwalletTransferMethodUi
+     */
+    public static synchronized HyperwalletTransferMethodUi getInstance(@NonNull final Context context,
+            @NonNull final HyperwalletAuthenticationTokenProvider authenticationTokenProvider) {
+        if (sInstance == null) {
+            sInstance = new HyperwalletTransferMethodUi();
```

Comment:
I think getInstance method should be like attached image as per confluence page section 3.2 1st diagram.
Also need to remove the old method or it need to be reuse instead of new method creation.
<img width="801" alt="Screenshot 2019-11-08 at 4 10 35 PM" src="https://user-images.githubusercontent.com/52773997/68470831-ddc51800-0242-11ea-8dd0-c1cac8441c1a.png">

Reply:
The inner code here is fairly similar to `HyperwalletInsight#initialize` (which is called below). And that method has the added benefit of calling it in a thread.

Reply:
Can you please confirm it. <img width="848" alt="Screenshot 2019-11-09 at 10 49 19 AM" src="https://user-images.githubusercontent.com/52773997/68523532-97be9180-02e0-11ea-8550-562bec905d1e.png">

Reply:
I also have and asked same @fmattos-hw 's doubt here. multiple getInstance needed?

Reply:
@manikandan-altmtrk thanks for pointing it out, it will be deprecated